src/cli/cli.cpp

```diff
@@ -620,6 +620,24 @@ template <> otError Interpreter::Process<Cmd("ba")>(Arg aArgs[])
                 otBorderAgentSetEphemeralKeyCallback(GetInstancePtr(), nullptr, nullptr);
             }
         }
+        /**
+         * @cli ba ephemeralkey feature (enable, disable)
+         * @code
+         * ba ephemeralkey feature
+         * Enabled
+         * Done
+         * ba ephemeralkey feature enable
+         * Done
+         * @endcode
+         * @par api_copy
```

Comment:
Can you add a parameter line for this one?  To show that `feature` is always required and `enable|disable` is optional

`@cparam ba ephemeralkey feature [@ca{enable|disable}]`